models/repo.go

```diff
@@ -1646,7 +1646,7 @@ func SearchRepositoryByName(opts *SearchRepoOptions) (repos []*Repository, _ int
 		}
 	}
 	if len(opts.Keyword) > 0 {
-		sess.And("repo.lower_name LIKE ?", "%"+strings.ToLower(opts.Keyword)+"%")
+		sess.Where("repo.lower_name LIKE ? OR repo.description LIKE ?", "%"+strings.ToLower(opts.Keyword)+"%", "%"+strings.ToLower(opts.Keyword)+"%")
```

Comment:
@Unknwon I haven't done so, or at least not sure if I have. How would I configure a test account to do so?

Looking at the xorm code `Where` returns for all intents and purposes an identical code path to `And`: https://github.com/gogits/gogs/blob/master/vendor/github.com/go-xorm/xorm/statement.go#L151

I have modified this pull request to replace my usage of `Where` with `And` (19c7bbb) and have tested that it still works as expected when searching for repositories by their name or by their description.